tests/test_wps_graph_tsstats.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Fri May 17 14:02:50 2019
+
+@author: ets
+"""
+
+from pywps import Service
+from pywps.tests import assert_response_success
+
+from raven.processes import GraphIndicatorAnalysis
+from .common import client_for, TESTDATA, CFG_FILE, get_output
+
+
+def test_graph_timeseries_stats(request):
+    client = client_for(Service(processes=[GraphIndicatorAnalysis(), ], cfgfiles=CFG_FILE))
+    datainputs = "ts_stats=files@xlink:href=file://{ts_stats};".format(ts_stats=TESTDATA['tsstats'])
+
+    datainputs = "ts_stats=files@xlink:href=file://{ts_stats};" \
```

Comment:
`datainputs` is redeclared here. Maybe have two tests?

Reply:
Forgot the first one, should have been removed. Fixed.